app/commands/welcome.js

```diff
@@ -0,0 +1,175 @@
+module.exports = Base => class extends Base {
+    constructor() {
+        super();
+        //this allows us to sort the welcome array in the player schema alphabetically and keep default as the first index.
+        this.welcomeOptions = {
+            'default': 'a',
+            'online': 'b',
+            'help': 'c',
+            'cooldowns': 'd',
+            'autostore': 'e',
+            'version': 'f',
+            'hints': 'z',
+            'reset': 'reset'
+        };
+
+        this.serverVersion = [
```

Comment:
For what purpose are we keeping a version and telling it to the user?
Versions are usually handled a lot different

Reply:
removed the serverVersion stuff. I've replaced it with a xp store option. the fact i didn't originally include that was an oversight on my part.